src/datepicker/examples/bootstrap.datepicker.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.datepicker.componenet:pfBootstrapDatepicker
+ * @element pf-bootstrap-datepicker
+ *
+ * @param {date} date must be a Javascript Date - to be displayed in the input. Can be left empty.
+ * @param {string} format default date format for displayed dates
+ * @param {object} dateOptions an object to configure the uib-datepicker in one place (optional)
```

Comment:
As this is an object, some reference material might be helpful. See other object descriptions which describe the format/syntax of the options allowed for date.